### DISCUSSION TITLE
You don't have permission to see the files on this site.

### DISCUSSION THREAD

**Discussion by Scippy** [dnnsoftware/Dnn.Platform#5088] [General]
In latest DNN 9.10.2 in some modules with Host user I'm notified with the message "You don't have permission to see the files on this site."

I've already tried to go to Site Assets, right click on Root and then set permissions and also perform a recursive sync. That has always worked for me but not this time.
I point out to you that the same modules work fine with DNN 9.10.1
I can't see any breaking change on latest release about site assets. Any idea to fix it?

**Comment by Scippy** [comment]
I've solved it adding Registered Users  with permission to browse and read in file asset.
This sound like a bug, I don't think that Host user should be tied to the permissions of registered users.

**Comment by wbonekamp** [comment]
Same problem here... we upgraded to 9.10.2 and in several modules we could upload files anymore.
Message pointed out it is a permission problem/issue: You don't have permission to.. etc.
Even if you login as host it shows the permission error

HTML editor didn't show subfolders when trying to upload an file (browse server)...

Yes it's a bug in DNN 9.10.2...

Workaround: manually set browse/read permissions on the folder in the Asset Manager.
Hopefully it''s fixed in 9.11.0

**Comment by Scippy** [comment]
it is necessary to make a clarification, the problem arises during the upgrade process, and it is possible to solve it only after having assigned the permissions through the site file asset on the portal with id 0 (the first site created on that installation).
Afterwards, the files for all the other sites of that DNN installation will also be visible again for the host user and all users with the right permissions.

**Reply by yog-it** [reply]
Could this be the same issue as #5095 ?
